src/rma/destruction/views.py

```diff
@@ -73,3 +78,24 @@ def form_valid(self, form):
         )
 
         return response
+
+
+class ReviewerDestructionListView(RoleRequiredMixin, FilterView):
+    """ data for user who can reviewer destruction lists"""
+
+    role_permission = "can_review_destruction"
+    template_name = "destruction/reviewer_list.html"
+    filterset_class = ReviewerListFilter
+
+    def get_queryset(self):
+        review_status = DestructionListReview.objects.filter(
+            author=self.request.user, destruction_list=models.OuterRef("id")
+        ).values("status")
+
+        prefiltered_qs = DestructionList.objects.filter(
+            assignee=self.request.user
+        ) | DestructionList.objects.reviewed_by(self.request.user)
+
+        return prefiltered_qs.annotate(
+            review_status=models.Subquery(review_status[:1])
+        ).order_by("-id")
```

Comment:
If there is a creation timestamp on the model, try to use that for sorting rather than the ID. IDs are not guaranteed to match to "most recent" first.

```suggestion
        ).order_by("-created")
```